css/css-basics_social-card/css/styles.css

```diff
@@ -1,3 +1,22 @@
 * {
   box-sizing: border-box;
 }
+img {
+  border-radius: 5px;
+}
+button {
+  border: none;
+  background-color: white;
+}
+article {
```

Comment:
There are several properties missing here: 
- background-color: white; => otherwise you can't see a difference to the body
- border-radius: 5px; => it makes the box round so that it looks like a card and is parallel to the image
- padding: 30px; => so there is enough space between the border of the box and the text within it

Reply:
background-color is white
i killed the border completely to make it look like in the original image
I have the 5px border radius for the image